direct/nn/unet/unet_2d.py

```diff
@@ -299,7 +299,7 @@ def unpad(
         w_mult: int,
     ) -> torch.Tensor:
 
-        return input_data[..., h_pad[0] : h_mult - h_pad[1], w_pad[0] : w_mult - w_pad[1]]
+        return input_data[..., int(h_pad[0]) : int(h_mult - h_pad[1]), int(w_pad[0]) : int(w_mult - w_pad[1])]
```

Comment:
Why was that necessary?

Reply:
This is necessary to convert to OpenVINO model from onnx.